frontend/components/PaidSelector.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div class="wrapper">
+    <div class="buttons noselect">
+      <button v-on:click="btn1Click" class="default btn1 activeBtnNo noselect">
+        BEZPŁATNE</button
+      ><button v-on:click="btn2Click" class="default btn2 noselect">
+        PŁATNE
+      </button>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  methods: {
+    btn1Click(e) {
+      e.preventDefault();
+      document.querySelector(".btn1").classList.add("activeBtnNo");
+      document.querySelector(".buttons").classList.add("bgColorRed");
+      document.querySelector(".btn2").classList.remove("activeBtnYes");
+    },
+
+    btn2Click(e) {
+      e.preventDefault();
+
+      document.querySelector(".btn2").classList.add("activeBtnYes");
+      document.querySelector(".buttons").classList.remove("bgColorRed");
+      document.querySelector(".buttons").classList.add("bgColorGreen");
+      document.querySelector(".btn1").classList.remove("activeBtnNo");
```

Comment:
1. Change methods name to more descriptive.
2. Why `querySelector`? You could do it by `computed` and conditional styles in Vue.

Reply:
Łukasz wasn't familiar with computed styles in vue. I'm already on it, thanks!

Reply:
Okay, don't know if You had exactly this in mind but I changed it and commited.

Reply:
@brzozka thanks. Yeah, i wasn't sure how to do this using computed styles so i used working method that i known. Anyway now it looks properly.